packit.yaml

```diff
@@ -1,14 +1,12 @@
 upstream_project_url: https://github.com/cockpit-project/cockpit-podman
 # enable notification of failed downstream jobs as issues
 issue_repository: https://github.com/cockpit-project/cockpit-podman
-specfile_path: cockpit-podman.spec
```

Comment:
See https://github.com/packit/packit-service/issues/1511 -- there's a bug that propose-downstream fails without an explicit `specfile_path:`, so we'll most likely have to bring that back anyway (with a bug ref). That will surface when testing the releasing, but until then we first need to sort out the `packit srpm` build.